.eslintrc.js

```diff
@@ -127,5 +128,8 @@ module.exports = {
     'import/resolver': {
       typescript: {},
     },
+    jsdoc: {
```

Comment:
Added because `.prettierrc.js` is now being linted (it was being ignored before, for some reason) and ESLint doesn't know what to do with the JSDoc by default.